src/foam/lib/json/AnyParser.java

```diff
@@ -18,6 +18,6 @@ public AnyParser() {
         new BooleanParser(),
         new LongParser(),
         new DoubleParser(),
-        new FObjectParser()));
+        new ExprParser()));
```

Comment:
So remove the existing Expr Parser, make a new parser for `__Property__` and then replace L:21 with `new Alt(new __PropertyParser__(), new FObjectParser());`?